packages/@lwc/engine-core/src/framework/hydration.ts

```diff
@@ -762,80 +823,47 @@ function validateStyleAttr(
     }
 
     if (!nodesAreCompatible) {
-        if (process.env.NODE_ENV !== 'production') {
-            const { getProperty } = renderer;
-            logWarn(
-                `Mismatch hydrating element <${getProperty(
-                    elm,
-                    'tagName'
-                ).toLowerCase()}>: attribute "style" has different values, expected "${vnodeStyle}" but found "${elmStyle}".`,
-                vnode.owner
-            );
-        }
+        queueHydrationError(HydrationError.attribute('style', `"${elmStyle}"`, `"${vnodeStyle}"`));
     }
 
     return nodesAreCompatible;
 }
 
-function areCompatibleStaticNodes(client: Node, ssr: Node, vnode: VStatic, renderer: RendererAPI) {
+function areStaticNodesCompatible(
+    clientNode: Node,
+    serverNode: Node,
+    vnode: VStatic,
+    renderer: RendererAPI
+) {
     const { getProperty, getAttribute } = renderer;
-    if (getProperty(client, 'nodeType') === EnvNodeTypes.TEXT) {
```

Comment:
IIUC none of this (782-800) would ever execute? `areCompatibleStaticNodes` is never called unless `serverNode (ssr)` is of type `Element` And the check for client can be moved to parent for symmetry?